src/utils/csv-parser-allowed-channels.js

```diff
@@ -0,0 +1,16 @@
+function parseAllowedChannels(channels) {
+  if (!channels) {
+    return [];
+  }
+
+  return channels
+    .split(';')
+    .filter(Boolean)
+    .map((channelData) => {
+      const [id, name] = channelData.split(',');
+
+      return { id, name };
+    });
+}
+
+module.exports = { parseAllowedChannels };
```

Comment:
maybe you can make more general this function changing its name from "parseAllowedChannels" to "parseCSVChannels"?

Reply:
It is a good idea but each csv have his own logic